### DISCUSSION TITLE
Should I use a personal access token or GCM, in the Publish to GitHub Packages module?

### DISCUSSION THREAD

**Discussion by rfalanga** [skills/.github#127] [Q&A] [answered]
I'm going through the GitHub Skill, currently doing _Publish to GitHub Packages_. At Step 4 the instructions say I'm to generate a personal access token. Following the [link in the instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token), however those instructions also say that it would be a better idea to use the [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) (GCM), if using GitHub from the command line. I am certain I have GCM installed.

For the purposes of this course, where I'm going everything in the browser on GitHub's website, which should I use?

**Comment by sinsukehlab** [comment, marked as answer]
The workflows in the course expect learners to use a personal access token.
You can try GCM and open an issue to share errors if any.

**Reply by sinsukehlab** [reply]
Thanks for Galaxy Brain.

**Reply by Pavlovskiiy** [reply]
gh_2.21.1_macOS_amd64 2

**Reply by Pavlovskiiy** [reply]
> Thanks for Galaxy Brain.
> 
>

**Reply by Pavlovskiiy** [reply]
> The workflows in the course expect learners to use a personal access token.
> 
> You can try GCM and open an issue to share errors if any.
> 
>
